tests/text-translator/en2gr-quantization.test

```diff
@@ -1,4 +1,4 @@
-RUN: %text-translator -m %models_dir/en2gr -keep-original-precision-for-nodes=Add -dump-profile=%t.yaml -quantization-schema=symmetric <<< "My name is Bob ." && \
+RUN: %text-translator -m %models_dir/en2gr -keep-original-precision-for-nodes=Add -dump-profile=%t.yaml -quantization-schema=symmetric <<< "My name is Bob ."
 RUN: (%text-translator -m %models_dir/en2gr -keep-original-precision-for-nodes=Add -load-profile=%t.yaml <<< "My name is Bob ." | %FileCheck %s)
```

Comment:
I am not sure why we need to pipe stuff into FileCheck. Isn't this obvious that we need to verify the output and compare it to the CHECK lines in this file?